exercises/practice/nucleotide-count/.docs/hints.md

```diff
@@ -0,0 +1,11 @@
+# Hints
+
+## General
+
+- The expected result is an `Association List` data structure (also called `alist`), not an string, or something else. You can find more information about it in this [link](https://www.gnu.org/software/emacs/manual/html_node/elisp/Association-Lists.html). So, for the correct example, the right output would be:
+
+```text
```

Comment:
Not 100% sure about this but since the documentation says
> The hints must appear as a Markdown list under a ## General heading.

I think the code block should read (indented with two spaces to be parsed as part of the list)

````markdown

  ```text
  "GATTACA" -> '((?A . 3) (?C . 1) (?G . 1) (?T . 2))
  ```
````

Reply:
Good catch, I'm going to fix it right now.